### PR TITLE
Disable subscription validation by default

### PR DESCRIPTION
**Pull request by netoneko**
It prevents us from accepting transactions on testnet

### CODE REVIEW

File: e2e/config/docker/docker-compose.test.services.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ services:
       service: base-service
     working_dir: /opt/orbs/projects/services/public-api-service-typescript/
     environment:
-      - validateSubscription=true
+      VALIDATE_SUBSCRIPTION: 'true'
```
Comment (lbeder) [inline e2e/config/docker/docker-compose.test.services.yml]: AFAIK, the quotes are not necessariy

File: projects/services/public-api-service-typescript/src/index.ts [hunk 2]
```diff
@@ -24,6 +24,10 @@ if (!NODE_NAME) {
 
 const nodeTopology = topology();
 const peers = topologyPeers(nodeTopology.peers);
-const nodeConfig = { nodeName: NODE_NAME };
+const validateSubscription = VALIDATE_SUBSCRIPTION === "true";
```
Comment (lbeder) [inline projects/services/public-api-service-typescript/src/index.ts]: You can also do `!!VALIDATE_SUBSCRIPTION`

File: e2e/config/docker/docker-compose.test.services.yml (hunk 1 above)
Comment (netoneko) [reply e2e/config/docker/docker-compose.test.services.yml]: I'm pretty sure without quotes it will fail on `docker-compose up`

Comment (lbeder) [reply e2e/config/docker/docker-compose.test.services.yml]: Really? `true` is a valid value in a YAML. Oh well.

**Pull request merged**